README.md

```diff
@@ -49,15 +41,27 @@ To build after all dependencies are installed:
     cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .
```

Comment:
`<LATEST_RELEASE_TAG>` has confused some people. Perhaps update it after every release? Or remove that line, then by default `master` branch will be used (but `--version` command won't show the correct tag).

Reply:
Agreed, let's make it simple for the end user to copy paste the script. I've added a new `current_version` file containing the text matching the (expected) release tag. I've swapped the <variable> for a $variable and inserted a bash script to set `variable=$(curl URL)`

Reply:
Removed the `curl` statement and `$variable`. Replaced with an explicit `git checkout master` and added a comment about replacing with current release as needed.